### PR TITLE
Add a Metron windows job

### PR DESCRIPTION
**Pull request by benmoss**
Hi Loggregators,

This PR adds a job for metron to run on Windows. We understand it's a little weird to ask you all to own this since BOSH Windows is still slightly in flux, but it's proving difficult to maintain all the releases for GardenWindows in separate release repos.

Happy to discuss!

Ben & Matt

### CODE REVIEW

**Comment by benmoss** [comment]
We're going to update this to compile the agent on a Windows compilation VM rather than using the Concourse job workflow we originally submitted

**Comment by benmoss** [comment]
Should be good to evaluate now!

File: config/blobs.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ---
-golang/go1.6.1.linux-amd64.tar.gz:
-  object_id: 221b054e-31a1-474a-ada6-00cc49578454
+golang/go1.5.4.windows-amd64.zip/go1.5.4.windows-amd64.zip:
+  object_id: 3905bc78-6893-4fdc-9010-186978dfb3b0
+  sha: 3cf7b469e08a68246e69a107e3ddbb5571b0919b
+  size: 86769706
+golang/go1.6.1.linux-amd64.tar.gz/c73492474dcc7a233772d27e948b74468d9f1849:
```
Comment (wfernandes) [inline config/blobs.yml]: Why is the original `go1.6.1.linux-amd64.tar.gz` blob not being used?

File: config/final.yml [hunk 2]
```diff
@@ -4,5 +4,5 @@ min_cli_version: 1.5.0.pre.1001
 blobstore:
   provider: s3
   options:
-    bucket_name: loggregator-release-blobs
+    bucket_name: loggregator-windows-bosh-blobs
```
Comment (wfernandes) [inline config/final.yml]: I'm not sure why our bucket name has to change?

**Comment by wfernandes** [comment]
Looks like there are conflicts in this PR.

**Comment by benmoss** [comment]
It looks like @aminjam and @bdshroyer are making changes to this PR unintentionally by pushing to this branch

**Comment by aminjam** [comment]
Yes we were trying to add golang windows blob to get this to work in our pipeline. If this is getting ready to be pulled in, we can rewrite the history and remove the blobstore changes.

**Comment by benmoss** [comment]
Will reopen this when we get a chance to clean things up.

**Comment by aminjam** [comment]
[PR 133](https://github.com/cloudfoundry/loggregator/pull/133) should be good to merge.

**Pull request closed without merging**